packages/components/src/Ratio.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+export const Ratio = ({ ratio, children }) => (
+  <div
+    css={{
+      width: '100%',
+      height: 0,
+      paddingBottom: ratio * 100 + '%',
```

Comment:
This sounds like a funny hack, however one would still need a way for "absolute positioning within the slide".

For instance:
1) Slide numbers are typically placed among the corners
2) Overlay captions